va/win32/va_win32.c

```diff
@@ -35,7 +35,7 @@
  * which will be selected when provided with an adapter LUID which
  * does not have a registered VA driver
 */
-const char VAAPI_DEFAULT_DRIVER_NAME[] = "vaon12";
+const char VAAPI_DEFAULT_DRIVER_NAME[] = "vaon12_drv_video";
```

Comment:
emmm. But "vaon12_drv_video" is not a full path. Full path (on Windows) would be for example "C:\msys64\mingw64\bin\vaon12_drv_video.dll". You probably can name it "full driver name", but saying "full path" I personally understand different thing.

Reply:
The full path should be `.\vaon12_drv_video.dll` intended as default for the [Nuget package](https://www.nuget.org/packages/Microsoft.Direct3D.VideoAccelerationCompatibilityPack/) that ships `vaon12_drv_video.dll` in the same directory as `va.dll`, `va_win32.dll`. 

As in Win `LoadLibrary` (aliased to `dlopen`) supports the file string to [have multiple semantics](https://learn.microsoft.com/en-us/windows/win32/api/libloaderapi/nf-libloaderapi-loadlibrarya#remarks) I only left the filename, which will first still try to open `.\vaon12_drv_video.dll` and then attempt again using the default Windows library search order.

- If no file name extension is specified in the lpFileName parameter, the default library extension .dll is appended. 
- When no path is specified, the function searches for loaded modules whose base name matches the base name of the module to be loaded. If the name matches, the load succeeds. Otherwise, the function searches for the file. The first directory searched is the directory containing the image file used to create the calling process...